nucleus/common/internal-api/src/main/java/fish/payara/internal/notification/PayaraNotification.java

```diff
@@ -117,20 +117,20 @@ protected void setMessage(String message) {
         this.message = message;
     }
 
-    public List<String> getWhitelist() {
-        return whitelist;
+    public List<String> getNotifierWhitelist() {
+        return notifierWhitelist;
     }
 
-    protected void setWhitelist(List<String> whitelist) {
-        this.whitelist = whitelist;
+    protected void setNotifierWhitelist(List<String> whitelist) {
+        this.notifierWhitelist = whitelist;
     }
 
-    public List<String> getBlacklist() {
-        return blacklist;
+    public List<String> getNotifierBlacklist() {
+        return notifierBlacklist;
     }
 
-    protected void setBlacklist(List<String> blacklist) {
-        this.blacklist = blacklist;
+    protected void setNotifierBlacklist(List<String> blacklist) {
+        this.notifierBlacklist = blacklist;
```

Comment:
I see the setters are still in and the builder is mostly a fluent API alias for calling setters on the notification. I really would like to see this using builder with fields for the individual values and a notification with a constructor accepting all the fields called on `build()`.